src/.gitignore

```diff
@@ -5,3 +5,5 @@
 .vscode/c_cpp_properties.json
 .vscode/launch.json
 __pycache__
+*.pyc
+defines_example.txt
```

Comment:
make this user_defines.txt

Reply:
Rename "user_defines_example.txt" to "user_defines.txt" and include all of the -D options but commented out.  This way new user can clearly see what is available and just needs to uncomment their Reg Domain and enter a UID.

Reply:
Yeah, fixed now.